src/api/spec/lib/feature_switch/feature_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Feature do
+  let(:user) { build(:confirmed_user, login: 'Tom') }
```

Comment:
Is there a reason for not creating this user?

Reply:
not, really

Reply:
changed